### ISSUE TITLE
Apache Config

### ISSUE THREAD

**Issue by spectravp** [bshaffer/oauth2-server-php#503]
This took me a while to figure out. I was able to successfully get an access token, but any other request using the access token would result in 403 forbidden. Turns out that on Apache 2.4 I had to add the code below to my config. Afterwards all worked fine. 

SetEnvIf Authorization "(.*)" HTTP_AUTHORIZATION=$1

**Comment by bshaffer** [comment]
We do have this issue outlined [here](https://github.com/bshaffer/oauth2-server-php/blob/develop/src/OAuth2/Request.php#L140), but I suppose we could add your method in there as well

**Comment by spectravp** [comment]
Ack! Wish I found that sooner. LOL. I never saw that that was documented inline in the code.

**Comment by bshaffer** [comment]
Yes, it's not super intuitive, and this is a very common issue people run into. Any suggestions on how to better clarify it?

**Comment by spectravp** [comment]
Of course, I would leave it inline in the code, but what about adding it to this page of the documentation under "Requirements" or "Installation"? http://bshaffer.github.io/oauth2-server-php-docs/

**Comment by bshaffer** [comment]
great idea!
